### PR TITLE
feat: separate ios and android celeryapp

### PR DESCRIPTION
**Pull request by alessandroberlati**
## Description

This PR separates the celery apps handling the authorization of an analytics token for iOS and the verification of a SafetyNet attestation for Android.

## Checklist

- [x] I have followed the indications in the [CONTRIBUTING](https://github.com/immuni-app/immuni-backend-analytics/blob/master/CONTRIBUTING.md).
- [x] The documentation related to the proposed change has been updated accordingly (plus comments in code).
- [x] I have written new tests for my core changes, as applicable.
- [x] I have successfully run tests with my changes locally.
- [x] It is ready for review! :rocket:

### CODE REVIEW

File: immuni_analytics/celery/authorization_android/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#   Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
```
Comment (ocardia) [inline immuni_analytics/celery/authorization_android/__init__.py]: I noticed an issue in the copyright plugin. As you can see it's adding `three` spaces after the `#`. If you check the template you gave, it should have `two`. What I did was changing the template to only have `one` (which is the original intention, btw) and re-run it.

The same applies to all the other copyrights added in this PR.

File: immuni_analytics/celery/authorization_android/tasks/verify_safety_net_attestation.py [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-#  it under the terms of the GNU Affero General Public License as
-#  published by the Free Software Foundation, either version 3 of the
-#  License, or (at your option) any later version.
-#  This program is distributed in the hope that it will be useful,
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-#  GNU Affero General Public License for more details.
-#  You should have received a copy of the GNU Affero General Public License
-#  along with this program. If not, see <https://www.gnu.org/licenses/>.
+#   Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
```
Comment (ocardia) [inline immuni_analytics/celery/authorization_android/tasks/verify_safety_net_attestation.py]: Here, as you can see, it's super clear.

**Review by ocardia — CHANGES_REQUESTED** [review]
Apart from the copyright, it all looks fine. 👌

File: immuni_analytics/celery/authorization_ios/__init__.py [hunk 3]
```diff
@@ -10,14 +10,3 @@
 #  GNU Affero General Public License for more details.
 #  You should have received a copy of the GNU Affero General Public License
 #  along with this program. If not, see <https://www.gnu.org/licenses/>.
-
-from immuni_common.models.enums import EnvarEnum
```
Comment (ocardia) [inline immuni_analytics/celery/authorization_ios/__init__.py:14]: If there are no other Enums in this file, it may be worth removing it altogether?

**Review by ocardia — APPROVED** [review]

File: immuni_analytics/celery/authorization_ios/__init__.py (hunk 3 above)
Comment (alessandroberlati) [reply immuni_analytics/celery/authorization_ios/__init__.py:14]: It's a weird ui problem :D It looks like it merged the deletion of enum.py with the creation of init.py

**Pull request merged**